### PR TITLE
Add Ephemery command to documentation

### LINKED ISSUE #1726: Add Documentation for Ephemery Testnet Implementation
## Description
The current documentation does not cover Ephemery Implementation.

## Where should this content be added?
The content will be added to the connect to a network -> connect to testnet section of the documentation. With this addition users can be aware that they have the option to run Ephemery network.

### PR DESCRIPTION
**Pull request by gconnect**
## Description
This PR add Ephemery commands to the following files:
- testnet.md
- run-docker-image.md
- bootnodes.md
- index.md
- network-and-chain-id.md
- node-sync.md
- start-node.md
- options.md
- besu-teku-testnet.md
- index.tsx

### Issue(s) fixed
 Fixes #1726

### Preview
https://besu-docs-git-fork-gconnect-add-documentatio-04d83e-hyperledger.vercel.app

### CODE REVIEW

**Comment by bgravenorst** [comment]
Thanks for this contribution @gconnect!
I've pinged someone to perform a tech review on this content.

I'll also review afterwards.

**Comment by gconnect** [comment]
> Thanks for this contribution @gconnect! I've pinged someone to perform a tech review on this content.
> 
> I'll also review afterwards.

Alright thanks for reviewing. I will be looking forward to your feedback.

File: docs/public-networks/get-started/start-node.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ To delete the local block data, delete the `database` directory in the `besu/bui
 
 ## Genesis configuration
 
-Besu specifies the genesis configuration, and sets the network ID and bootnodes when connecting to [Holesky](#run-a-node-on-holesky-testnet), [Sepolia](#run-a-node-on-sepolia-testnet), and [Mainnet](#run-a-node-on-ethereum-mainnet).
+Besu specifies the genesis configuration, and sets the network ID and bootnodes when connecting to [Holesky](#run-a-node-on-holesky-testnet), [Sepolia](#run-a-node-on-sepolia-testnet),[Ephemery](#run-a-node-on-ephemery-testnet) and [Mainnet](#run-a-node-on-ethereum-mainnet).
```
Comment (macfarla) [inline docs/public-networks/get-started/start-node.md:27]:
```suggestion
Besu specifies the genesis configuration, and sets the network ID and bootnodes when connecting to [Holesky](#run-a-node-on-holesky-testnet), [Sepolia](#run-a-node-on-sepolia-testnet), [Ephemery](#run-a-node-on-ephemery-testnet) and [Mainnet](#run-a-node-on-ethereum-mainnet).
```

File: docs/public-networks/get-started/start-node.md [hunk 2]
```diff
@@ -104,6 +104,18 @@ Where `<path>` and `<sepoliadata-path>` are the path and directory to save the S
...
+
+To run a node on [Ephemery](https://github.com/ephemery-testnet/ephemery-resources?tab=readme-ov-file) specifying a data directory:
+
+```bash
+besu --network=ephemery --data-path=<path>/<ephemery-data-path>
+```
+
+Where `<path>` and `<ephemery-data-path>` are the path and directory to save the Ephemery chain data to.
+
+See the [guide on connecting to a testnet](connect/testnet.md) for more information.
```
[the remark this answers is not in the document]
Comment (gconnect) [reply docs/public-networks/get-started/start-node.md:117]: > this line is repeated from 10 lines above - maybe that's ok

Yeah, that was how the other testnets were outlined. I just made a copy for ephemery.

**Comment by macfarla** [comment]
LGTM from content POV just prob some formatting / linting to review @bgravenorst

File: src/components/HomepageCards/index.tsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ const CardList: CardItem[] = [
     // prettier-ignore
     description: (
       <>
-        Run Besu as an execution client on Ethereum Mainnet and Ethereum public testnets, such as Holesky and Sepolia.
+        Run Besu as an execution client on Ethereum Mainnet and Ethereum public testnets, such as Holesky, Ephemery and Sepolia.
```
Comment (bgravenorst) [inline src/components/HomepageCards/index.tsx:29]:
```suggestion
        Run Besu as an execution client on Ethereum Mainnet and Ethereum public testnets, such as Holesky, Ephemery, and Sepolia.
```

**Review by bgravenorst — COMMENTED** [review]
Looks good, thanks! All minor feedback :)
- We generally try and keep [line lengths short](https://docs-template.consensys.io/contribute/format-markdown#column-limit)
- We use the serial comma (I fixed these inline)
- It appears a table may be missing a pipe

**Comment by gconnect** [comment]
@bgravenorst I just made the requested changes.

**Review by bgravenorst — APPROVED** [review]
Approved. Thanks again for the submission @gconnect.

**Pull request merged**